.github/workflows/dusk-only.yml

```
@@ -0,0 +1,139 @@
+name: Dusk Browser Tests Only
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)